cypress/plugins/index.js

```diff
@@ -51,10 +51,17 @@ module.exports = (on, config) => {
 
   const waitUntilAppRestarts = async (timeout) => await waitOn({ delay: 2000, resources: [config.baseUrl], timeout })
   const getReplacementText = async (text, source) => source ? fsp.readFile(source) : text
-  const replaceText = ({ text, originalText, newText, source }) => getReplacementText(newText, source)
-    .then((replacementText) => {
-      return text.replace(originalText, replacementText)
-    })
+  const replaceText = ({ text, originalText, newText, source }) => {
+    return getReplacementText(newText, source)
+      .then((replacementText) => {
+        if (text.indexOf(originalText) > -1) {
```

Comment:
Can we change this to `text.includes(originalText)` please?

Reply:
I can

Reply:
Done